### DISCUSSION TITLE
500 Internal Server Error

### DISCUSSION THREAD

**Discussion by drdan18** [livewire/livewire#5374] [1. Help]
I have a user of my web application getting a 500 Internal Server Error when submitting a POST from a form. 

I can not reproduce this error on my computer, phone, or tablet.  Nor with my phone connection or home internet connection.  This user can reproduce the error on phone, computer, and when using a different internet connection. 

User has tried with Firefox and Chrome browser and mobile phone browser.  

There is no further detail to the nature of the error.  The Network Activity from DevTools just states 500 Internal Server Error.  

Any ideas how I can find more info about the nature of this error?  

Dan

**Comment by Martleby** [comment]
I'd suggest checking the laravel.log - it should contain the details of the error.

Cheers,
Martin.

**Comment by CrazyBoy49z** [comment]
> I have a user of my web application getting a 500 Internal Server Error when submitting a POST from a form.
> 
> I can not reproduce this error on my computer, phone, or tablet. Nor with my phone connection or home internet connection. This user can reproduce the error on phone, computer, and when using a different internet connection.
> 
> User has tried with Firefox and Chrome browser and mobile phone browser.
> 
> There is no further detail to the nature of the error. The Network Activity from DevTools just states 500 Internal Server Error.
> 
> Any ideas how I can find more info about the nature of this error?
> 
> Dan 

Upload  Photo?

**Comment by alfakiat** [comment]
There are some errors occur while running the project.
to determine what is error you can try one of these:

1. in `php.ini` file enable `display_errors`
2. from laravel `.env` file set `APP_DEBUG=true`
3. see `storage/logs/laravel.log` that contains a log of errors
